src/clients/DclData.ts

```diff
@@ -36,7 +44,7 @@ export type TransparencyData = {
     total: bigint
     budget: bigint
   }
-  teams: Team[]
+  teams: Committee[]
```

Comment:
```suggestion
  commitees: Committee[]
```
Shouldn't we update Transparency so it is using the "commitee" name there instead of team?